src/core/LabeledSelect/LabeledSelect.tsx

```diff
@@ -42,6 +42,11 @@ export type LabeledSelectProps<T> = {
    * @default 'default'
    */
   displayStyle?: 'default' | 'inline';
+  /**
+   * If true, shows a red asterisk but does not prevent form submission.
+   * @default false
+   */
+  required?: boolean;
```

Comment:
Since this is different from how `LabeledInput` and `LabeledTextarea` work, I explicitly mentioned that it does not prevent form submission.

Reply:
Created a discussion #67 so we can address the root cause in the future